features/step_definitions/networking.rb

```diff
@@ -696,3 +685,35 @@
     raise "Fail to delete the ovs pod"
   end
 end
+
+Given /^the default interface on nodes is stored in the#{OPT_SYM} clipboard$/ do |cb_name|
+  ensure_admin_tagged
+  step "I select a random node's host"
+  node_name ||= node.name
+  cb_name = "interface" unless cb_name
```

Comment:
`||=` operator means to set the value in case variable is `false` or `nil`. Thus above code will be nicer as:
```
node_name = node.name
cb_name ||= "interface"
```